packages/emulation/libretro-picodrive/package.mk

```
@@ -44,6 +44,7 @@ post_configure_target() {
 }
 
 make_target() {
+  unset R
   make -f Makefile.libretro
```

```suggestion
  R= make -f Makefile.libretro
```
I don't where the `R` is coming from but rather than unsetting it completely (which could have consequences later) wouldn't it be better to just unset it when only calling `make`?

---

R seems to be leaking in from jenkins build scripts, so far we had the build error only on jenkins builds.

But `R= make ...` works as well, no objections to use that